backend/api/models.py

```
@@ -181,6 +181,30 @@
+    filename = models.CharField(max_length=255)
+    recording_date = models.DateTimeField(auto_now_add=True)
+    upload_status = models.CharField(max_length=20, choices=UPLOAD_STATUS_CHOICES, default='pending')
+    upload_completed_at = models.DateTimeField(null=True, blank=True)
+    recording_url = models.URLField(max_length=500, blank=True, null=True)
+    
+    class Meta:
+        ordering = ['-recording_date']
+        verbose_name = 'Webcam Recording'
+        verbose_name_plural = 'Webcam Recordings'
+    
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)